core/src/main/java/hudson/model/FingerprintCleanupThread.java

```diff
@@ -66,13 +70,13 @@ private static FingerprintCleanupThread getInstance() {
     public void execute(TaskListener listener) {
         int numFiles = 0;
 
-        File root = new File(Jenkins.getInstance().getRootDir(),"fingerprints");
-        File[] files1 = root.listFiles(LENGTH2DIR_FILTER);
+        File root = new File(getRootDir(), FINGERPRINTS_DIR_NAME);
```

Comment:
nit: To me it feels like it would make more sense to have a method `File getFingerPrintsFile()` for testing rather than separating out just `String getRootDir()`.